### ISSUE TITLE
UI Enhancement,  Need to add functionality in buttons like Animation or Vibration effects 

### ISSUE THREAD

**Issue by therealsanjeev** [systers/powerup-android#1369]
## Description
When the user clicked on the any button and locked/unlocked scenario in our game they receive feedback for a better experience, she does not receive any message or any feedback for the user's satisfaction. we add this for a better experience.

#### For Locked Scenario -
when I'm trying to click on the locked scenario, nothing will happen and I'm getting confused, that the game is properly working or not although screen showing it is locked, as a user human satisfaction is more important.
#### Any Button -
after the clicking, button vibrates his phone for a better experience. this should be added on the whole game buttons.
that would be a great UX and also user feels a good experience.

I need to do,
When the user clicks on the Locked scenario
1.) Display a message for the user to know they've pressed on a scenario
2.) Vibrate the phone

When the user clicked on any button on the game
1.)  Vibrate the phone

## Mocks
![Screenshot (38)](https://user-images.githubusercontent.com/40147174/75865334-689cd480-5e29-11ea-95a6-5cae8df69b53.png)

### Update [Required]
For Locked Scenario
- [ ] MapActiviy.java
- [ ] MapLevel2.java

### Enhancement to Update [Optional]
- [ ] better experience or feedback

## Definition of Done
- [ ] All of the required items are completed.
- [ ] Approval by 1 mentor.

## Estimation
5 hour

**Comment by sidvenu** [comment]
Hey Sanjeev. Welcome to AnitaB.org Open Source! :)

I just have a few quick suggestions in raising issues that you can incorporate to make it better.

Whatever is in the brackets [ ], those are placeholders that you should replace. The intention is to make a template everyone follows so that issues remain structured.

Instead of "add a message using Toast
() or Snackbar()", you can write "Display a message for the user to know they've pressed on a scenario". Same goes with your other point - you can write "Vibrate the phone".

You can also include the above points in Adceptance Criteria, rather than having the file names in the criteria.

And you don't need to include the code to accomplish this in the description. That can be there in the commit you do :)

Regarding your UX suggestion itself, I don't see a reason why a message should be displayed to the user when they click on a scenario. Did you have anything particular in mind when you thought about it?

That said, I think including a vibration feedback when the user presses only on a locked scenario to let them know it's locked. To me, I don't think including it in unlocked scenarios would improve the UX.

Also, if you're a GSoC aspirant, I think instead of creating an issue here for improving an existing thing, you can include this in your proposal for improving PowerUp. The admins are having a hard time keeping track of the influx of issues due to GSoC aspirants thinking they've to contribute code to improve their chances.

Do let me know what you think about my suggestions. Good work! :)

**Comment by pknotfound** [comment]
You'd also need to add to the manifest file and make sure that you add additional checks for SDK Version to avoid any new warnings coming your way.

**Comment by therealsanjeev** [comment]
@sidvenu , thanks bro to giving time to my issue, this is my first issue that's why I made lots of mistakes. I'll keep in mind your all suggestions for the future.

now come back to my issue, as a user when I'm trying to click on the locked scenario, nothing will happen and I'm getting confused, that the game is properly working or not although screen showing it is locked, as a user human satisfaction is more important. that why I raise this issue.
your point of view is right but if we show a small message, that would be a great UX and also user feels a good experience.
Thanks to supporting my issue:)

**Comment by therealsanjeev** [comment]
> You'd also need to add to the manifest file and make sure that you add additional checks for SDK Version to avoid any new warnings coming your way.

thanks bro for your tip :)

**Comment by pknotfound** [comment]
> > You'd also need to add to the manifest file and make sure that you add additional checks for SDK Version to avoid any new warnings coming your way.
> 
> thanks bro for your tip :)

Happy to help.
You can use this thread if you face any issue.

**Comment by pknotfound** [comment]
@therealsanjeev Hello, have you taken up this issue?

**Comment by therealsanjeev** [comment]
@Priyansh-Kedia yes bro, before that I want to confirm this issue by mentor'S. any mentor, whenever you feel free please look into this :)

**Comment by pknotfound** [comment]
Alright.
All the best.
